docs/content/dagster-cloud/account/managing-users/managing-teams.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Managing teams in Dagster Cloud | Dagster Docs
+---
+
+# Managing teams in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud Enterprise.</Note>
+
+As part of our role-based access control (RBAC) feature, Dagster Cloud supports the ability to assign users to teams. A team is a group of users with a set of default deployment, code location, and Branch Deployment user roles.
+
+In this guide, we'll cover how to add, manage, and remove teams in Dagster Cloud.
+
+---
+
+## Prerequisites
+
+To use this feature, you'll need a **Dagster Cloud Enterprise plan.**
+
+---
+
+## Adding teams
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users/managing-user-roles-permissions#teams">
+    Organization Admin permissions
+  </a>{" "}
+  are required to add teams.
+</Note>
+
+1. In the Dagster Cloud UI, click the **user menu (your icon) > Cloud Settings**.
+2. Click the **Teams** tab.
+3. Click the **Create a team** button.
+4. In the window that displays, enter a name in the **Team name** field.
+5. Click **Create team**.
+
+After the team is created, you can [add users and assign user roles to deployments](#managing-teams).
+
+---
+
+## Managing teams
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users/managing-user-roles-permissions#teams">
+    Organization Admin permissions
+  </a>{" "}
+  are required to manage teams.
+</Note>
+
+In the **Cloud Settings > Teams** tab:
+
+1. Locate the team you want to modify in the table of teams.
+2. Click the **Edit** button in the team's row.
+
+From here, you can [manage team members](#managing-team-members) and [the team's roles for deployments](#managing-team-roles).
+
+### Managing team members
+
+**Note**: Organization Admin permissions are required to add or remove team members.
+
+#### Adding team members
+
+1. In the **Members** tab, use the search bar to locate a user in your organization.
+2. Once located, click the user.
+3. Click **Add user to team**.
+4. Repeat as needed, clicking **Done** when finished.
+
+#### Removing team members
+
+1. In the **Members** tab, locate the user in the list of team members.
+2. Click **Remove from team**.
+3. Repeat as needed, clicking **Done** when finished.
+
+### Managing team roles
+
+**Note**: Admin users can only modify team roles in deployments where they are an Admin.
```

Comment:
extreme nitpick but i think it's *for* deployments not *in* deployments? in makes me think you need to be logged into that deployment in the UI

Reply:
Fair enough!